### DISCUSSION TITLE
Set Storageclass for Volume Snapshots

### DISCUSSION THREAD

**Discussion by Rohmilchkaese** [vmware-tanzu/velero#7764] [Community support Q&A] [answered]
Hi @All,

i have a question. I cant find anything in documentation about it. 
Is it possible to storageclass for Volume snapshots?

Background: Lets say i have a storage class which is standard. That standard storage class always does 3 replicas for pvs.
Which is nice in terms of safety. But when using Velero to create Backups of pvs content, with volume snapshots, that means that the snapshots also gets replicated. Which in my opinion doesn't make sense. 
So i thought, okay, i just create a storage class with no replicas and pass that to velero. 
It would first of all speed things up, in my case. And also safe a lot of storage space. And third theres no reason in replicating pvs which velero create while doing volume snapshots. Just get it done and upload it. 

Am I on the right track with my assumptions?

**Comment by draghuram** [comment, marked as answer]
I assume you meant PVC created from volume snapshot rather than snapshot itself. There is already an enhancement request open for this: https://github.com/vmware-tanzu/velero/issues/7700.

**Reply by Rohmilchkaese** [reply]
Yes, exactly! Ah nice, thanks! I havent found it. Nice!
